tools/engine_tool/lib/src/gn_utils.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:process_runner/process_runner.dart';
+
+import 'environment.dart';
+import 'proc_utils.dart';
+
+/// Information about a test build target.
+final class TestTarget {
+  /// Construct a test target.
+  TestTarget(this.label, this.executable);
+
+  /// The build target label.
```

Comment:
Super nit: this is a bit abstract, a concrete example might be useful:

```suggestion
  /// The build target label. Example: `//flutter/fml`
```

I don't feel strongly about this, feel free to skip

Reply:
Done.